docs/tutorials/best-practices/speed-up-templates.md

```diff
@@ -0,0 +1,143 @@
+# Speed up your Coder templates and workspaces
+
+October 31, 2024
+
+---
+
+If it takes your workspace a long time to start, find out why and make some
+changes to your Coder templates to help speed things up.
+
+## Monitoring
+
+You can monitor [Coder logs](../../admin/monitoring/logs.md) through the
+system-native tools on your deployment platform, or stream logs to tools like
+Splunk, Datadog, Grafana Loki, and others.
+
+### Workspace build timeline
+
+Use the **Build timeline** to monitor the time it takes to start specific
+workspaces. Identify long scripts, resources, and other things you can
+potentially optimize within the template.
+
+![Screenshot of a workspace and its build timeline](../../images/best-practice/build-timeline.png)
+
+Adjust this request to match your Coder access URL and workspace:
+
+```shell
+curl -X GET https://coder.example.com/api/v2/workspacebuilds/{workspacebuild}/timings \
+  -H 'Accept: application/json' \
+  -H 'Coder-Session-Token: API_KEY'
+```
+
+Visit the
+[API documentation](../../reference/api/builds.md#get-workspace-build-timings-by-id)
+for more information.
+
+### Coder Observability Chart
+
+Use the [Observability Helm chart](https://github.com/coder/observability) for a
+pre-built set of monitoring integrations.
+
+With the Observability Helm chart, you can monitor [which specific startup
+metrics are a part of the Helm chart?]
+
+Visit the
+[observability repository](https://github.com/coder/observability?tab=readme-ov-file#installation)
```

Comment:
```

We reccommend that all administrators deploying on Kubernetes set the observability bundle up with the control plane from the start. For installation instructions, you can visit the [observability repository](https://github.com/coder/observability?tab=readme-ov-file#installation), or our [Kubernetes installation guide](../../install/kubernetes.md).

```

Reply:
I'm into this + we should add it to the [kubernetes doc](https://coder.com/docs/install/kubernetes#4-install-coder-with-helm)

which, aside, something looks busted here. Can I tweak the MD?

![2024-10-31_14-49-19](https://github.com/user-attachments/assets/4a66c72f-c02b-4ebe-914c-8fb1e3c857ed)

Reply:
I believe the observability bundle used to be in the k8s install docs. Unsure where it went :(

Yeah, if you want to fix the numbers, go for it. This is definitely an annoyance in our parser.

Reply:
- [markdown's fixed there](https://coder.com/docs/@bp-speed-up-templates/install/kubernetes?__vercel_draft=1#4-install-coder-with-helm)

  ![2024-10-31_15-22-05](https://github.com/user-attachments/assets/66acf5f4-087a-4fe0-8a69-e7ed190236cd)

- and as a [separate commit](https://github.com/coder/coder/pull/15296/commits/62deabc03e5b2494cf9b36720c7813fccb9be380)(for easy reverting if we need), I added a slightly tweaked version of your suggested text to the [k8s install doc (preview)](https://coder.com/docs/@bp-speed-up-templates/install/kubernetes?__vercel_draft=1#coder-observability-chart). 


- :shipit: on ✅  ?